File: micro-digiservices/ecreditservice/src/main/java/io/digiservices/ecreditservice/dto/InfoAdministrative.java
package io.digiservices.ecreditservice.dto;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import io.digiservices.clients.domain.AgenceDto;
import io.digiservices.clients.domain.DelegationDto;
import io.digiservices.clients.domain.PointVenteDto;
import io.digiservices.clients.domain.User;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@JsonIgnoreProperties(ignoreUnknown = true)
public class InfoAdministrative {

    private User user;
    private DelegationDto delegationDto;
    private AgenceDto agenceDto;
    private PointVenteDto pointVenteDto;
}

File: frontend/ultima-ng-19.0.0/src/app/interface/infoAdministrative.ts
import { Agence } from './agence';
import { Delegation } from './delegation';
import { PointVente } from './point.vente';
import { IUser } from './user';

export interface InfoAdministrative {
    delegationDto: Delegation;
    user: IUser;
    pointVenteDto: PointVente;
    agenceDto: Agence;
}
